src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -73,6 +73,16 @@ def sql_alter_from(self, catalog):
             f" TBLPROPERTIES ('upgraded_from' = '{self.key}');"
         )
 
+    def sql_unset_to(self, catalog):
+        return (
+            f"ALTER {self.kind} `{catalog}`.`{self.database}`.`{self.name}` "
+            f"UNSET TBLPROPERTIES IF EXISTS('upgraded_to');"
+        )
+
+    # SQL to reset the assessment record to revert migration state
+    def sql_unset_to_assessment(self, schema):
+        return f"UPDATE {schema}.tables SET upgraded_to=NULL where DATABASE = {self.database} and NAME = {self.name}"
```

Comment:
Why database and name columns are uppercase?

Reply:
Fixed